src/checkout-buttons/strategies/paypalCommerce/paypalCommerce-button-options.ts

```diff
@@ -0,0 +1,6 @@
+export interface PaypalCommerceButtonInitializeOptions {
+    /**
+     * The Client ID of the Paypal App
+     */
+    clientId: string;
```

Comment:
Do you require the caller to pass the client ID in? I thought it'd be a part of the `PaymentMethod` object. If so, you shouldn't need to pass it from the outside because the button strategy itself should have access to the `PaymentMethod` object and read the required field.

Reply:
Thank you!
Actually, I'm not sure. I did it like 'PayPal Express' and I think it needs only for paypal commerce buttons. And it has sent during initialization buttons
But I think it's not part of PaymentMethod

Reply:
Right I see, indeed it's not getting returned. I wonder if we can add that field to our API so we don't have to pass it as a template variable. I know for Amazon we're passing `clientId` under `initializationData`. But it's not ideal because `initializationData` is a schema-less field. Perhaps we can make it a top-level field instead? cc @BC-ACherednichenko

Reply:
@davidchin, yep, it looks much better to pass `client-id` through `PaymentMethod`, we will check it out